### PR TITLE
GitHomeworkFixToErrors homework

### PR DESCRIPTION
**Pull request by DJMF2015**
Glasgow-Scotland Class-5 - David Fulton - Git/Github intro. - Week 2 [Intro Module]

# Your Details

- Your Name: David Fulton
- Your City: Edinburgh based
- Your Slack Name: David F.

# Homework Details

- Module: Intro to Github/Git
- Week: 2

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -4,110 +4,108 @@
...
 
-You send the changes of your local repository to Github
+Uploading the changes or modifications made to a local repository to the remote server usually on github. 
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+A distributed version control system for developers
```
Comment (sSalem123) [inline homework.md:88]: Hi David, i think the question 14 the right answer  git is  version control not for github

**Comment by MaarcisO** [comment]
Good work David!

File: homework.md (hunk 1 above)
Comment (DJMF2015) [reply homework.md:88]: > Hi David, i think the question 14 the right answer git is version control not for github

Hi Thanks for pointing this out. I will amend my answer accordingly!

**Comment by DJMF2015** [comment]
Amended incorrections.

**Pull request closed without merging**